.github/workflows/modules-check.yml

```diff
@@ -94,7 +94,7 @@ jobs:
         uses: taiki-e/install-action@cargo-hack
       - name: cargo hack
         working-directory: ./modules
-        run: cargo hack --feature-powerset check --lib --tests
+        run: cargo hack check --feature-powerset --mutually-exclusive-features wynd,osmosis,astroport,bow,terraswap,astrovault --lib --tests
```

Comment:
Hm `testing,osmosis-test,interface,cw-orch` features is mutually exclusive to those as well, or no? It turns it down to `144` combinations

Reply:
Can we make our own cargo hack sets? 

I'd say we just need to compile with
- All chains, no cw-orch
- All chains, with cw-orch
- No chains, no cw-orch
- No chains, with cw-orch

where we replace "with cw-orch" with the biggest feature that includes cw-orch (so probably osmosis-test?)

Reply:
Yeah that's great!

Reply:
> Yeah that's great!

Ah it tries to use `group` features even if there's none of these features.
This flag with "include-features" didn't help:
```
--ignore-unknown-features
    Skip passing --features flag to `cargo` if that feature does not exist in the package.

    This flag can only be used together with either --features or --include-features.
```
As it still tries to include group everywhere

I'll just mutually exclude it then as originally planned, don't think it makes sense to create cargo hack command for every existing package